### PR TITLE
Adding non-searachble dropdown to the Style Guide

### PR DESCRIPTION
**Pull request by tejans24**
connects #1243

### CODE REVIEW

**Comment by NickHeiner** [comment]
We can look at this after #1870 goes in.

File: client/app/components/SearchableDropdown.jsx [hunk 1]
```diff
@@ -113,6 +114,7 @@ class SearchableDropdown extends Component {
         placeholder={placeholder === null ? DEFAULT_PLACEHOLDER : placeholder }
         clearable={false}
         noResultsText={noResultsText ? noResultsText : NO_RESULTS_TEXT}
+        searchable={_.isNil(searchable) ? true : searchable}
```
Comment (mdbenjam) [inline client/app/components/SearchableDropdown.jsx]: Can we not just pass searchable?

File: client/app/containers/StyleGuide/StyleGuideSearchableDropdown.jsx [hunk 2]
```diff
@@ -37,11 +37,23 @@ class StyleGuideSearchableDropdown extends Component {
     return (
       <div>
         <StyleGuideComponentTitle
-          title="Searchable Dropdowns"
+          title="Dropdown Menus"
```
Comment (mdbenjam) [inline client/app/containers/StyleGuide/StyleGuideSearchableDropdown.jsx]: Did this change break the sidebar link?

File: client/app/components/SearchableDropdown.jsx (hunk 1 above)
Comment (tejans24) [inline client/app/components/SearchableDropdown.jsx]: I wanted to pass true by default; if searchable is not passed.

File: client/app/containers/StyleGuide/StyleGuideSearchableDropdown.jsx (hunk 2 above)
Comment (tejans24) [inline client/app/containers/StyleGuide/StyleGuideSearchableDropdown.jsx]: The link is fixed now

**Pull request merged**